hazelcast/src/main/java/com/hazelcast/core/DistributedObjectEvent.java

```diff
@@ -25,11 +25,11 @@
  */
 public class DistributedObjectEvent {
 
-    private EventType eventType;
+    protected EventType eventType;
```

Comment:
this is dangerous is it effectively makes the fields to be part of API. I'd avoid it if possible.